src/utils/EventHandlers.tsx

```diff
@@ -1,20 +1,21 @@
 import * as Config from "./Config";
-import * as Types from "./Types";
+import * as Renderers from "./Renderers";
 import * as StateUpdaters from "./StateUpdaters";
+import * as Types from "./Types";
 
 /////////////////
 // MapChart.tsx
 /////////////////
 
-export const handleMoveEnd = (stateManager: Types.stateManager) => (
-  newPosition: Types.position
+export const handleMoveEnd = (stateManager: Types.StateManager) => (
+  newPosition: Types.Position
 ): void => {
   const [state, dispatch] = stateManager;
   const originalPosition = state.mousePosition;
   const zoomLevelChanged = newPosition.zoom !== originalPosition.zoom;
-  return zoomLevelChanged
+  zoomLevelChanged
     ? dispatch({
-        type: "updateMousePositionAndRefreshMarkers",
```

Comment:
Related to "Persisted row/marker highlight/details for combinedRows, including ones that split/merge upon zooming in/out"
I no longer reset combinedRows (now allCombinedRows) and currentCombinedRow (now currentCombinedRows) to their default values after updating the mouse position, which may happen after the mouse coordinates or zoom change.